.github/workflows/build-and-deploy-staging.yaml

```
@@ -0,0 +1,79 @@
+          token: ${{ secrets.GIT_TOKEN }}
+
+      - uses: unfor19/install-aws-cli-action@master
+      - name: AWS CLI Config
+        run: |
+          mkdir -p ~/.aws
+          aws configure set aws_access_key_id ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws configure set aws_secret_access_key ${{secrets.AWS_SECRET_ACCESS_KEY }}
+          aws configure set default.region ${{ secrets.AWS_DEFAULT_REGION }}
+          aws ecr get-login-password --region ${{ secrets.AWS_DEFAULT_REGION }} | docker login --username AWS --password-stdin 953966107292.dkr.ecr.us-east-1.amazonaws.com
+      - name: Build and push
+        uses: docker/build-push-action@v3
```

actionlint (1.7.7)

the runner of "docker/build-push-action@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)